src/app/AppContainer.js

```diff
@@ -1,14 +1,23 @@
 import { compose } from 'redux'
 import { withRouter } from 'react-router'
 import { connect } from 'react-redux'
-import { App } from './App'
+
+import { getCurrentUser } from 'store/thunks/users'
 import { maintenanceSelector } from 'store/selectors/maintenanceSelector'
 
+import { App } from './App'
+
 export function mapStateToProps(state) {
   return {
     modalOpen: state.modal.isActive,
     isMaintenanceActivated: maintenanceSelector(state),
   }
 }
 
-export default compose(withRouter, connect(mapStateToProps))(App)
+export function mapDispatchToProps(dispatch) {
+  return {
+    getCurrentUser: () => dispatch(getCurrentUser()),
+  }
+}
```

Comment:
```suggestion
const mapDispatchToProps = { getCurrentUser }
```
Droit au but et conseillé par react-redux